core/templates/dev/head/pages/forum/forum.html

```diff
@@ -20,13 +20,11 @@
     }
   </style>
 
-  <div class="container">
+  <div class="container" ng-controller="Forum">
     <div class="oppia-forum">
```

Comment:
Let's move the `ng-controller` in this div, would make it more specific, right ?

Reply:
Done!!